static/app/views/replays/detail/useVirtualizedList.tsx

```diff
@@ -7,14 +7,23 @@ type Opts = {
   ref: RefObject<List>;
 };
 function useVirtualizedList({cellMeasurer, deps, ref}: Opts) {
-  const cache = useMemo(() => new CellMeasurerCache(cellMeasurer), [cellMeasurer]);
+  const cache = useMemo(() => {
+    return new CellMeasurerCache(cellMeasurer);
+  }, [cellMeasurer]);
 
   const updateList = useCallback(() => {
     cache.clearAll();
     ref.current?.forceUpdateGrid();
   }, [cache, ref]);
 
   // Restart cache when items changes
+  // XXX: this has potential to break the UI, especially with dynamic content
+  // in lists (e.g. ObjectInspector). Consider removing this as deps can easily
+  // be forgotten to be memoized.
+  //
+  // The reason for high potential to break UI: updateList clears the cache, so
+  // any cells that were expanded but scrolled out of view will have their
+  // cached heights reset while they re-render expanded.
```

Comment:
This is the reason for all of the above `useMemo` calls: we were creating a new array of `deps` on each re-render *and* new `items` from `useConsoleFilters` and `useVirtualizedList`.

I'm thinking we may want to remove this `useEffect` as the potential to set it off is high.